website/src/documentation/sdks/.htaccess

```diff
@@ -0,0 +1,23 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Javadocs / pydocs are available only on the published website, published from
+# https://github.com/apache/beam-site/tree/release-docs
+# They were previously hosted within this repository, and published at the URL
+# path /documentation/sdks/(javadoc|pydoc)/..
+# The following redirect maintains the previously supported URLs.
+RedirectMatch permanent "/(javadoc|pydoc)" "https://beam.apache.org/documentation/releases/$1"
```

Comment:
Caveat: I have virtually no prior experience with these files.
This looks to me like a redirect loop, since the rule matches both old and new paths containing `(javadoc|pydoc)`.

I would write it like this:
```
RedirectMatch permanent "^/documentation/sdks/(.*)" "/documentation/releases/$1"
```

ref: https://httpd.apache.org/docs/2.4/mod/mod_alias.html#redirectmatch

Reply:
Thanks for calling this out; I went through the trouble of installing a local apache server to test this, and found the current version didn't work at all. I've updated it and tested that the current version now works.

Note that I've also moved the redirect rule previously in https://github.com/apache/beam-site/blob/release-docs/.htaccess to this file, since that one won't work as-is; I'll remove it.

Reply:
See: https://github.com/apache/beam-site/pull/571